### PR TITLE
Show icon in authorize screen

### PR DESCRIPTION
**Pull request by joshenlim**
Demo:
image

### CODE REVIEW

File: studio/pages/authorize.tsx [hunk 1]
```diff
@@ -176,10 +176,13 @@ const APIAuthorizationPage: NextPageWithLayout = () => {
             <div className="rounded-md border border-scale-600 p-2.5 flex items-center">
               <div
                 className="w-8 h-8 md:w-10 md:h-10 bg-center bg-no-repeat bg-cover flex items-center justify-center"
-                // [Joshen] For when we support icons
-                // style={{ backgroundImage: `url('${requester?.icon}')` }}
+                style={{
+                  backgroundImage: requester?.icon !== null ? `url('${requester?.icon}')` : 'none',
```
Comment (alaister) [inline studio/pages/authorize.tsx:180]: Is the `?` still needed in `url('${requester?.icon}')`?

**Review by alaister — APPROVED** [review]
LGTM!

File: studio/pages/authorize.tsx (hunk 1 above)
Comment (joshenlim) [reply studio/pages/authorize.tsx:180]: yeap good point, shouldn't need now

**Comment by MildTomato** [comment]
For future consideration - would it make sense to move to a _bullet point_ style for listing the permissions being granted.

Screenshot 2023-05-19 at 12 16 58 PM

older github example
image

**Pull request merged**